omod/src/main/webapp/fragments/patientdashboard/visits.gsp

```diff
@@ -57,12 +73,39 @@
             encounterCount = ${ param.encounterCount }; // This variable is defined in patientdashboard/patientDashboard.gsp
         <% } %>
         loadTemplates(visitId, ${ patient.id }, fromEncounter, encounterCount);
+
+        <%
+            def visits = patient.allVisitsUsingWrappers;
+            visits.eachWithIndex{ wrapper, idx ->
```

Comment:
do we really have to iterate through all visits for this? Shouldn't the currently-displayed visit already be available in a variable somewhere?